src/cgr_gwas_qc/models/config/user_files.py

```diff
@@ -89,6 +89,12 @@ class UserFiles(BaseModel):
         description="The full path to an aggregated FAM file if the sample level GTC files are not available.",
     )
 
+    # BCF
+    bcf: Optional[Path] = Field(
+        None,
+        description="The full path to an aggregated BCF/VCF file perferably encoding the GenCall scores.",
+    )
+
```

Comment:
Please add a validator for BCF. Use the `validate_gtc_pattern` as a template.

Reply:
Added validator in 52f97c790a3d3fb43ec28167757e21cf95ca71a2 to ensure the input `bcf `ends in `.bcf`